Modules/posixmodule.c

```diff
@@ -12836,7 +12836,7 @@ os.scandir
 
 Return an iterator of DirEntry objects for given path.
 
-path can be specified as either str, bytes or path-like object.  If path
+path can be specified as either str, bytes or a path-like object.  If path
```

Comment:
For consistency with the use of the oxford comma (the comma before the "or") we should use it here as well:

```suggestion
path can be specified as either str, bytes, or a path-like object.  If path
```

Reply:
(If you apply this suggestion using GitHub, remember to regenerate the clinic file)